main.go

```diff
@@ -175,6 +175,11 @@ func preRun(cmd *cobra.Command, args []string) error {
 		viper.SetDefault("mssql.sslmode", "true")
 		viper.SetDefault("mssql.port", 1433)
 		required = append(required, "schema")
+	case "crdb":
```

Comment:
Can we have these directly in the driver? As the above TODO states, need to move them into the drivers anyway at some point.

Reply:
It'll also affect breaking this one out.